cmd/cluster/read-replica/read_replica.go

```diff
@@ -363,35 +363,24 @@ var deleteReadReplicaCmd = &cobra.Command{
 			if returnStatus != "SUCCEEDED" {
 				logrus.Fatalf("Operation failed with error: %s", returnStatus)
 			}
-			fmt.Printf("All Read Replica has been deleted for cluster %s.\n", formatter.Colorize(clusterName, formatter.GREEN_COLOR))
+			fmt.Printf("All Read Replica has been deleted for cluster %s.\n", formatter.Colorize(ClusterName, formatter.GREEN_COLOR))
 			return
 		}
-		fmt.Printf("All Read Replica has been deleted for cluster %s.\n", formatter.Colorize(clusterName, formatter.GREEN_COLOR))
+		fmt.Printf("All Read Replica has been deleted for cluster %s.\n", formatter.Colorize(ClusterName, formatter.GREEN_COLOR))
 
 	},
 }
 
 func init() {
 	ReadReplicaCmd.AddCommand(getReadReplicaCmd)
-	getReadReplicaCmd.Flags().StringVarP(&clusterName, "cluster-name", "c", "", "[REQUIRED] The name of the cluster.")
```

Comment:
Why cluster is not mandatory anymore ?

Reply:
I made it a persistent flag in cluster.go

```golang
	ClusterCmd.AddCommand(readreplica.ReadReplicaCmd)
	readreplica.ReadReplicaCmd.PersistentFlags().StringVarP(&readreplica.ClusterName, "cluster-name", "c", "", "[REQUIRED] The name of the cluster.")
	readreplica.ReadReplicaCmd.MarkPersistentFlagRequired("cluster-name")
```